src/casp/services/api/schemas/cell_operations.py

```diff
@@ -22,18 +22,46 @@ class DevDetailObject(BaseModel):
     mean_distance: float = Field(example=1791.372802734375)
 
 
-class MatchInfoDevDetails(MatchInfo):
+class AnnotationInfoCellTypeCountDevDetail(AnnotationInfoCellTypeCount):
```

Comment:
`NeighborhoodSummaryCellTypeStatiticsExtended`.  I was never a fan of `DevDetail`.  While you're at it, please rename those too.